src/themes/material/templates/edit-cell-bool.tpl.html

```diff
@@ -3,7 +3,7 @@
 		<md-checkbox
 			q-grid-focus="input"
 			[(ngModel)]="$view.edit.cell.value"
-			[q-grid-blur]="$view.edit.cell.commit.execute">
+			[input-on-blur]="$view.edit.cell.commit.execute">
```

Comment:
all common directives and components should start with `q-grid`

Reply:
according to this remark, the directive has been renamed to 'q-grid-on-blur' in the commit: [blur directive, dom base class: refactoring](https://github.com/qgrid/ng2/pull/39/commits/bb9b43a5374e93ada17ace5c6f028fd5aa7ac651).